tests/scripts/remote_monitoring_tests/rest_apis/test_update_results.py

```diff
@@ -180,9 +180,11 @@ def test_update_results_with_missing_metrics_section(test_name, result_json_file
         for err in error_data:
             actual_error_message = err["message"]
             if test_name == "Missing_metrics_bulk_res_few_containers" and d["interval_end_time"] == "2023-04-13T23:29:20.982Z":
-                error_message = "kubernetes_objects : Metric data is not present for container : tfb-server-1 for experiment: quarkus-resteasy-kruize-min-http-response-time-db. Metric data is not present for container : tfb-server-0 for experiment: quarkus-resteasy-kruize-min-http-response-time-db"
+                error_message = "Performance profile: [Metric data is not present for container : tfb-server-1 for experiment: quarkus-resteasy-kruize-min-http-response-time-db. ] , " \
+                                "Performance profile: [Metric data is not present for container : tfb-server-1 for experiment: quarkus-resteasy-kruize-min-http-response-time-db. , " \
```

Comment:
Why is tfb-server-0 changed to tfb-server-1 here?

Reply:
Reverted now. It wasn't required.